azure/master-arm/byol-upi/mainTemplate-dev.json

```diff
@@ -211,19 +211,32 @@
       "type": "string",
       "metadata": {
         "description": "Indicate whether you want to receive emails that contain the connection details and credentials for the Suite. These emails are sent using the provided SMTP server details."
-      },
+      }
+    },
+    "privateCluster": {
       "defaultValue": "false",
       "allowedValues": [
         "true",
         "false"
-      ]
+      ],
+      "type": "String",
+      "metadata": {
+        "description": "Select true to create a private OpenShift cluster."
+      }
     },
     "openShiftClustervnetId": {
-      "type": "string",
+      "defaultValue": "kxr601-vnet",
```

Comment:
vnet name is supposed to be hardcoded?
kxr601-vnet

Reply:
Will remove this all these harcoding , Thanks